lambda-deployment-orca/src/main/java/com/amazon/aws/spinnaker/plugin/lambda/traffic/LambdaTrafficUpdateVerificationTask.java

```diff
@@ -63,7 +73,47 @@ public TaskResult execute(@Nonnull StageExecution stage) {
             return formErrorTaskResult(stage,op.getErrors().getMessage());
         }
 
+        if (!DeploymentStrategyEnum.$WEIGHTED.toString().equals(stage.getContext().get("deploymentStrategy"))) {
+            boolean inValid = validateWeights(stage);
+            if (inValid) {
```

Comment:
This doesn't seem to be covered by tests. I think it's a good idea to have a test case for this as well.

Reply:
done